tests/testthat/test_pipelines.R

```diff
@@ -107,3 +107,69 @@ test_that("survtoregr 3", {
   expect_prediction_surv(p)
   expect_true("distr" %in% p$predict_types)
 })
+
+skip_if_not_installed("mlr3learners")
+
+test_that("survtoclassif", {
+  requireNamespace("mlr3learners")
+
+  pipe = mlr3pipelines::ppl("survtoclassif", learner = lrn("classif.log_reg"))
+  expect_class(pipe, "Graph")
+  grlrn = mlr3pipelines::ppl("survtoclassif", learner = lrn("classif.log_reg"),
+                             graph_learner = TRUE)
+  expect_class(grlrn, "GraphLearner")
+  grlrn$train(task)
+  p = grlrn$predict(task)
+  expect_prediction_surv(p)
+
+  cox = lrn("surv.coxph")
+  suppressWarnings(cox$train(task))
+  p2 = cox$predict(task)
+
+  expect_equal(p$truth, p2$truth)
+  expect_equal(p$score(), p2$score(), tolerance = 0.1)
+
+  # Test with cut
+  grlrn = mlr3pipelines::ppl("survtoclassif", learner = lrn("classif.log_reg"),
+                            cut = c(10, 30, 50), graph_learner = TRUE)
+  expect_class(grlrn, "GraphLearner")
+  suppressWarnings(grlrn$train(task))
+  p = grlrn$predict(task)
+  expect_prediction_surv(p)
+
+  # Test with max event time
+  max_time = task$data()[status == 1, min(time)]
```

Comment:
Did some refactoring in the tests and noticed this: here you want `max` or `min`?

Reply:
I want `min` here, since `max_time` needs to be greater than the minimum event time.